pylib/System/TestDef.py

```diff
@@ -437,21 +437,20 @@ def configTest(self):
             if not os.path.isfile(testFile):
                 print("Test .ini file not found!: " + testFile)
                 sys.exit(1)
-##TO DO, remove from loop, change testFile to list of files               
-            self.config = configparser.ConfigParser()
-            self.config.read(testFile)
-            for section in self.config.sections():
-                if self.logger is not None:
-                    self.logger.verbose_print("SECTION: " + section)
-                    self.logger.verbose_print(self.config.items(section))
-                if self.options['dryrun']:
-                    continue
-                if section.startswith("SKIP") or section.startswith("skip"):
-                    # users often want to temporarily ignore a section
-                    # of their test definition file, but don't want to
-                    # remove it lest they forget what it did. So let
-                    # them just mark the section as "skip" to be ignored
-                    continue;
+        self.config = configparser.ConfigParser()
+        self.config.read(self.log['inifiles'])
```

Comment:
self.config.read(testFile) was part of the loop checking to see if the .ini files existed.  I took the self.config.read out of the loop and used the list of files from the new argparse args.ini_files which was producing a list of lists so I needed to add the [0] to get to the actual list of files. ConfigParser will take either a single file or a list of files but it will not take a list of lists.